### DISCUSSION TITLE
Is there a way to add a user with "bot" in their name to not be blocked?

### DISCUSSION THREAD

**Discussion by seanenriquez** [baffler/Transparent-Twitch-Chat-Overlay#306] [General]
My twitch name has the word "bot" and my messages are blocked from the streamer's overlay chat. Doesn't seem like there's a way to let my messages through without completely removing all bot chats from their overlay.

**Comment by Ludiks** [comment]
I could greatly use this also.
Streambot relay from Restream doesnt show up and I could see messages from both instead of twitch only :(
